tests/componenttests/server/tests/sessionServerStateChange/SessionServerStateChangeTest.cpp

```diff
@@ -44,6 +44,55 @@ class SessionServerStateChangeTest : public RialtoServerComponentTest
         ASSERT_TRUE(receivedMessage);
         EXPECT_EQ(receivedMessage->sessionserverstate(), ::rialto::SessionServerState::INACTIVE);
     }
+
+    void setStateActive()
+    {
+        ::rialto::SetStateRequest request{createSetStateRequest(::rialto::SessionServerState::ACTIVE)};
+
+        ExpectMessage<::rialto::StateChangedEvent> expectedMessage(m_serverManagerStub);
+
+        ConfigureAction<::firebolt::rialto::server::ct::SetStateRequest>(m_serverManagerStub).send(request).expectSuccess();
+
+        auto receivedMessage = expectedMessage.getMessage();
+        ASSERT_TRUE(receivedMessage);
+        EXPECT_EQ(receivedMessage->sessionserverstate(), ::rialto::SessionServerState::ACTIVE);
+    }
+
+    void setStateInactive()
+    {
+        ::rialto::SetStateRequest request{createSetStateRequest(::rialto::SessionServerState::INACTIVE)};
+
+        ExpectMessage<::rialto::StateChangedEvent> expectedMessage(m_serverManagerStub);
+
+        ConfigureAction<::firebolt::rialto::server::ct::SetStateRequest>(m_serverManagerStub).send(request).expectSuccess();
+
+        auto receivedMessage = expectedMessage.getMessage();
+        ASSERT_TRUE(receivedMessage);
+        EXPECT_EQ(receivedMessage->sessionserverstate(), ::rialto::SessionServerState::INACTIVE);
+    }
+
+    void setLogLevels()
+    {
+        ::rialto::SetLogLevelsRequest request{createSetLogLevelsRequest()};
+
+        ConfigureAction<::firebolt::rialto::server::ct::SetLogLevelsRequest>(m_serverManagerStub)
+            .send(request)
+            .expectSuccess();
+    }
+
+    void sendAndRecievePing()
+    {
+        ::google::protobuf::int32 kMyId{3};
+        ::rialto::PingRequest request{createPingRequest(kMyId)};
+
+        ExpectMessage<::rialto::AckEvent> expectedMessage(m_serverManagerStub);
+
+        ConfigureAction<::firebolt::rialto::server::ct::PingRequest>(m_serverManagerStub).send(request).expectSuccess();
+        auto receivedMessage = expectedMessage.getMessage();
+        ASSERT_TRUE(receivedMessage);
+        EXPECT_EQ(receivedMessage->id(), kMyId);
+        EXPECT_EQ(receivedMessage->success(), true);
+    }
 };
 
 TEST_F(SessionServerStateChangeTest, ShouldConfigureInInactiveState)
```

Comment:
All tests should have a detailed description, please follow the template https://www.stb.bskyb.com/confluence/display/2016/Test+Description+Template